apps/02RockPaperScissors.js

```diff
@@ -8,17 +8,39 @@ prompt.start();
 function rockPaperScissors(hand1, hand2) {
   // Write code here
   if (hand1 === hand2) {
-    return "It's a tie"
+    return "It's a tie!";
   }
+
   else if (hand1 === 'rock') {
     if (hand2 === 'scissors') {
-      return 'Hand one wins!'
+      return 'Hand one wins!';
     }
     else {
-      return 'Hand two wins!'
+      return 'Hand two wins!';
     }
   }
 
+   else if (hand1 === 'paper') {
+     if (hand2 === 'rock') {
+       return 'Hand one wins!';
+     }
+     else {
+       return 'Hand two wins!';
+     }
+   }
+
+   else {
+     if (hand2 === 'paper') {
```

Comment:
Sound logic here. 

You could also combine all winning combinations into one 'else if' statement using the **||** operator.